x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -448,6 +448,27 @@ filebeat.inputs:
         value: '[[now (parseDuration "-1h")]]'
 ----
 
+[[response-force_ndjson]]
+[float]
+==== `response.force_ndjson`
+
+If the Content-Type header is set to "application/x-ndjson" then the ndjson results will be available in a new array called "ndjson-results".
```

Comment:
I would add a comment about supporting ndjson at the top of the docs instead for visibility.